### PR TITLE
Improve the setup script

### PR DESCRIPTION
**Pull request by hiraku-wfs**
You can check build as follows:

```sh
$ mkdir emlinux
$ cd emlinux/
$ mkdir repos
$ git clone -b pr-setup-script-2 https://github.com/hiraku-wfs/meta-emlinux.git repos/meta-emlinux
$ SYNC_POKY=1 source repos/meta-emlinux/scripts/setup-emlinux build
$ echo "MACHINE = \"qemuarm64\"" >> conf/local.conf
$ bitbake core-image-minimal
```

### CODE REVIEW

File: scripts/setup-emlinux [hunk 1]
```diff
@@ -1,32 +1,62 @@
...
 if [ -n "$1" ]; then
     BUILDDIR=$1
 fi
 
-sudo apt-get update
-sudo apt-get install gawk wget git-core diffstat unzip texinfo gcc-multilib \
-     build-essential chrpath socat cpio python python3 python3-pip python3-pexpect \
-     xz-utils debianutils iputils-ping libsdl1.2-dev xterm
+REPOS=${PWD}/repos
+if [ ! -e ${REPOS}/poky ]; then
```
Comment (masami256) [inline scripts/setup-emlinux]: The "-e" expression doesn't care about given path is file or directory or other type, so  "-d" would be nice to check directory exists or not.

File: scripts/setup-emlinux [hunk 2]
```diff
@@ -1,32 +1,62 @@
...
+    git clone -b master https://git.yoctoproject.org/git/poky ${REPOS}/poky
+fi
+if [ ! -e ${REPOS}/meta-debian ]; then
+    git clone -b master \
+	https://github.com/meta-debian/meta-debian.git ${REPOS}/meta-debian
+fi
+if [ ! -e ${REPOS}/meta-debian-extended ]; then
+    git clone -b master \
+	https://github.com/iwamatsu/meta-debian-extended.git \
```
Comment (masami256) [inline scripts/setup-emlinux]: Can we change meta-debian-extended repo from Iwamatsu-san's to miraclelinux's?

Comment (hiraku-wfs) [reply scripts/setup-emlinux]: Yes. I will use miraclelinux's repository.

File: scripts/setup-emlinux (hunk 1 above)
Comment (hiraku-wfs) [reply scripts/setup-emlinux]: Yes. I will use "-d".

**Pull request closed without merging**